client/src/components/Register.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import { Input } from '@nextui-org/react';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import backgroundImage from '../img/register-splash.jpg';
+import { setUser } from '../redux/user/userSlice';
+
+export const Register = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  const register = async (e) => {
+    e.preventDefault();
+    if (username.length >= 8 && password.length >= 8) {
+      dispatch(setUser({ username, password }));
+      localStorage.setItem('user', JSON.stringify({ username, password }));
+      navigate('/');
+    } else {
+      <p>Username and Password must be at least 8 characters long</p>;
+    }
```

Comment:
- [ ] The reservations that are being fetched by the server are the same for all users.  Kindly, make it so each user has their own reservations.

That way, your app will be consistent, and more useful.  👍

Reply:
Thanks for the update, we have fixed this issue now! ✅